mergify_engine/worker.py

```diff
@@ -355,6 +359,21 @@ async def consume(self, stream_name: StreamNameType) -> None:
         )
         LOG.debug("cleanup stream end", stream_name=stream_name)
 
+    async def _refresh_merge_trains(self, installation):
+        async with self._translate_exception_to_retries(
+            installation.stream_name,
+        ):
+            for train_name in await installation.redis.keys(
+                merge_train.Train.get_redis_key_for(installation.owner_id, "*", "*")
```

Comment:
nitpick: would be cleaner to have a `Train.iter_trains(owner_id)` class method that would return an iterator over all the train availables.

Reply:
I like it!